### PR TITLE
Bump to v0.100.5-dev1

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: .bumpversion_client.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.100.5-dev0
+current_version = v0.100.5.dev1
```
Comment (palango) [inline .bumpversion_client.cfg]: should the `v` be here?

Comment (LefterisJP) [reply .bumpversion_client.cfg]: no. Let me fix this. Why did this happen ? :(

**Comment by pirapira** [comment]
Looks very similar to #4484.  The only thing I don't know is whether the changelog is complete.

**Comment by LefterisJP** [comment]
> Looks very similar to #4484. The only thing I don't know is whether the changelog is complete.

Should be. We have the rule of updating the changelog for all user facing changes at the PR. If it's not then it's an ommision by the PR writer/reviewer, but that can be also added later.

**Pull request merged**